packages/react/package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "@emotion/core",
-  "version": "11.0.0-next.9",
-  "main": "dist/core.cjs.js",
-  "module": "dist/core.esm.js",
+  "name": "@emotion/react",
+  "version": "11.0.0-next.7",
+  "main": "dist/react.cjs.js",
+  "module": "dist/react.esm.js",
   "browser": {
-    "./dist/core.cjs.js": "./dist/core.browser.cjs.js",
-    "./dist/core.esm.js": "./dist/core.browser.esm.js"
+    "./dist/react.cjs.js": "./dist/react.browser.cjs.js",
```

Comment:
this example shows a little bit that maybe preconstruct should include the scope in generated filenames?

Reply:
Yeah, definitely something to consider